src/components/Subscribe/Subscribe.tsx

```diff
@@ -63,11 +86,18 @@ const Subscribe = ({
     >
       <h3 className={`${px}-subscribe__title`}>{title}</h3>
       {blurb ? <p className={`${px}-subscribe__blurb`}>{blurb}</p> : null}
+      {isLoading ? <p className={`${px}-subscribe__loading`}>{loadingText}</p> : null}
+
       <Input
         className={`${px}-subscribe__input`}
         type="email"
```

Comment:
TBH, I really don't like the look of the OOTB browser validation and would rather just do our own invalid state validation

Reply:
you can see the desired error states here: https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5730-11728&t=Nl1MOE1dSPwIxfrS-4